ui/widgets/layout/src/components/layout-widget.tsx

```diff
@@ -196,20 +201,30 @@ const Layout: React.FC<RootComponentProps> = props => {
                     </WarningCard>
                   )}
 
-                  <WarningCard margin={{ bottom: 'small' }} pad="small" direction="row">
-                    <WarningIcon type="error" themeColor="secondary" />
-                    <Box width="100%">
-                      <Text size="medium">
-                        {`${t('Following the merge, the Rinkeby network has been deprecated')}. ${t(
-                          'We have migrated Ethereum World to the Goerli testnet',
-                        )}. ${t('This will not affect your content or posts, they are saved')}! ${t(
-                          'But some functionalities such as claiming ENS names won’t be possible',
-                        )}. ${t('We are working hard on mitigating any issues')}. ${t(
-                          'Bear with us 🙏🏽',
-                        )}.`}
-                      </Text>
-                    </Box>
-                  </WarningCard>
+                  {!closeNotification && (
+                    <WarningCard margin={{ bottom: 'small' }} pad="small" direction="row">
+                      <WarningIcon type="error" themeColor="secondary" />
+                      <Box width="100%">
+                        <Text size="medium">
+                          {`${t(
+                            'Following the merge, the Rinkeby network has been deprecated',
+                          )}. ${t('We have migrated Ethereum World to the Goerli testnet')}. ${t(
+                            'This will not affect your content or posts, they are saved',
+                          )}! ${t(
+                            'But some functionalities such as claiming ENS names won’t be possible',
+                          )}. ${t('We are working hard on mitigating any issues')}. ${t(
+                            'Bear with us 🙏🏽',
+                          )}.`}
+                        </Text>
+                      </Box>
+                      <Icon
+                        type="close"
+                        clickable={true}
+                        onClick={() => setCloseNotification(true)}
```

Comment:
this method should be extracted, otherwise you're going to create a new function on every rerender